src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -308,8 +308,10 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 			if (element.nodeName.toLowerCase() === 'table') {
 				// add table header and table rows.
 				children.push(element.children[0]);
```

Comment:
>element.children[0 [](start = 18, length = 18)

should we also add a check for children before accessing the first ?

Reply:
Not a bad idea at all. Added a quick check.